apps/sms/views/conversation/style/conversation.css

```diff
@@ -157,3 +157,11 @@
 #messages-header-text .ellipsis-dir-fix {
   width: auto;
 }
+
+.conversation-view .panel {
```

Comment:
In my navigation rewrite, I set a "panel-active" class on the panel section that is currently active...

With the current code I don't have a better idea though.